src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -27,18 +27,16 @@
 
 <section name="Quick Start for Cassandra backend">
 
-  <p>The goal of the document is to allow anyone to start with James on Docker as an operational mail server.</p>
-  <p>The method described bellow should not be used in production.</p>
-
-  <p>Alternatively a <a href="https://github.com/apache/james-project/blob/9502cfebfd9a4eaebbba7dda802d1fe0ad3589b7/dockerfiles/run/docker-compose.yml">
-    docker-compose</a> is available for Guice + Cassandra + ElasticSearch</p>
+  <p>The goal of the document is to allow anyone to start a James instance on Docker as an operational mail server.</p>
+  <p>The two methods described bellow should not be used in production.</p>
```

Comment:
No, it's a quick start, it's only a reminder, IMHO.
A quick start should never be used in production, and our customers know that.